scripts/db/src/login.ts

```diff
@@ -120,75 +101,11 @@ async function login(
       }
     }
 
-    async function loginParfitUser(): Promise<CallbackUser | null> {
```

Comment:
A lot of the code being removed here is also duplicated in `scripts/db/src/get-user.ts` so it probably also needs to be removed there as well. I guess we can remove `pg` as a dependancy in `package.json` too.

Reply:
It actually seems rather important that we *don't* remove it from get-user. IDK, it depends on your point of view. But its presence in get-user prevents users from signing up and seeing someones donation details from a long time ago.

Reply:
Yes, I agree. 
It's important that we keep it for get-user for exactly that reason. Thanks for pointing it out JP